notebooks/STIS/custom_ccd_darks/custom_ccd_darks.ipynb

```diff
@@ -0,0 +1,559 @@
+{
```

Comment:
<p>Hi Jackie,</p><p>Running this notebook locally and have noticed that it references the <code>CRDS</code> cache. To ensure a successful run of the CI, I recommend setting the necessary environment variables, just as it is done with the ccd_darks notebook. Also this would require of adding the <code>crds</code> nto<code>requirements.txt</code> as well as installing right reference files.<code>!crds beststref --update-bestrefs --sync ......</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/hst_notebooks/pull/52/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>